stencil-library/src/components/card-form/example.js

```diff
@@ -1,53 +1,6 @@
-<!DOCTYPE html>
-<html dir="ltr" lang="en">
-
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
-  <title>card-form component</title>
-
-  <script type="module" src="../../build/webcomponents.esm.js"></script>
```

Comment:
We used to use these files locally for development, but now that they are just code samples we should update the src to be the cdn link pointing to `latest`

Reply:
This code is being deleted anyways, since for example usage, it's not relevant how you decided to include the files, that's described in the Introduction page.